bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HHresonant/Spin-0/Radion_hh_narrow_M900_proc_card.dat

```diff
@@ -0,0 +1,8 @@
+set group_subprocesses Auto 
+set ignore_six_quark_processes False 
+set gauge unitary 
+set complex_mass_scheme False 
+import model heft_radion 
+define p = g u c d s u~ c~ d~ s~ 
+generate p p > h2, ( h2 > H H ) 
```

Comment:
here you require h2 to be on-shell, this is different from the Run2 setup "generate p p > h2 > H H", it's already discussed?

Reply:
There may have been different versions of spin-0/2 X->HH gridpacks generated for Run 2. AFAIK the on-shell syntax is the one that was used by the analyses formerly part of the HIG PAG.

Reply:
@menglu21 since we are assuming NWA, the two syntax should be equivalent, but if you think `p p > h2 > H H` would be better (maybe for "generality" if one wants to introduce width later on), I'm happy to make the change.

Reply:
ok, It's fine generally, my concern is for the mass point M=250, whether these two syntaxs will be different or not, could you please have a check on that

Reply:
![image](https://github.com/cms-sw/genproductions/assets/72452043/d537e76e-f84c-43c8-aa62-79cd191089cb)
There is indeed some very small difference here (mX histograms for Spin-2 resonances), but for all practical purpose it should be negligible

Reply:
thanks, I'll merge this PR